src/components/common/item-selector/item-selector.js

```diff
@@ -62,8 +62,10 @@ const DropdownWrapper = styled.div`
   z-index: ${props => props.theme.dropdownWrapperZ};
   position: absolute;
   bottom: ${props => (props.placement === 'top' ? props.theme.inputBoxHeight : 'auto')};
-  margin-top: ${props => (props.placement === 'bottom' ? '4px' : 'auto')};
-  margin-bottom: ${props => (props.placement === 'top' ? '4px' : 'auto')};
+  margin-top: ${props =>
+    props.placement === 'bottom' ? `${props.theme.dropdownWapperMargin}px` : 'auto'};
```

Comment:
`${props.theme.dropdownWapperMargin}px`

**'px'** should be part of _dropdownWapperMargin_

Reply:
actually, most of the sizes in kepler theme don't have a specified unit